scripts/deploy-introspection-schema.sh

```diff
@@ -0,0 +1,24 @@
+# Taking a compressed introspection schema file and uploading it to S3
+# This is specifically for s3://graphiq.medplum.com/schema/
+
+version=$1
+client_id=$2
+client_secret=$3
+cors_file=$4
+
+curl 'http://localhost:8103/fhir/R4/$graphql' \
+  -u $client_id:$client_secret \
+  -H 'Content-Type: application/json' \
+  --data-raw '{"query":"    query IntrospectionQuery {      __schema {                queryType { name }        mutationType { name }        subscriptionType { name }        types {          ...FullType        }        directives {          name          description                    locations          args {            ...InputValue          }        }      }    }    fragment FullType on __Type {      kind      name      description            fields(includeDeprecated: true) {        name        description        args {          ...InputValue        }        type {          ...TypeRef        }        isDeprecated        deprecationReason      }      inputFields {        ...InputValue      }      interfaces {        ...TypeRef      }      enumValues(includeDeprecated: true) {        name        description        isDeprecated        deprecationReason      }      possibleTypes {        ...TypeRef      }    }    fragment InputValue on __InputValue {      name      description      type { ...TypeRef }      defaultValue                }    fragment TypeRef on __Type {      kind      name      ofType {        kind        name        ofType {          kind          name          ofType {            kind            name            ofType {              kind              name              ofType {                kind                name                ofType {                  kind                  name                  ofType {                    kind                    name                  }                }              }            }          }        }      }    }  ","operationName":"IntrospectionQuery"}' \
+  > schema-$version.json
+
+gzip schema-$version.json
+
+aws s3api put-bucket-cors \
```

Comment:
Is this a one-time thing? Or do we need to call this everytime we update the file?

If it's a one time thing, then maybe we should pull this out of the script entirely.

Reply:
It's a one-time thing